exim/README.md

```diff
@@ -0,0 +1,64 @@
+# Agent Check: exim
+
+## Overview
+
+This check monitors [exim][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][3] for guidance on applying these instructions.
+
+### Installation
+
+To install the exim check on your host:
+
+
+1. Install the [developer toolkit][10] on any machine.
+
+2. Run `ddev release build exim` to build the package.
+
+3. [Download the Datadog Agent][2].
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/exim/dist/<ARTIFACT_NAME>.whl`.
```

Comment:
Once this integration is released, there's no need to download the developer toolkit. Customers will just be able to install the integration with instructions [from here](https://docs.datadoghq.com/agent/guide/use-community-integrations/?tab=agentv721v621).